### DISCUSSION TITLE
Github classroom autograding edit button is broken

### DISCUSSION THREAD

**Discussion by marknelsonengineer** [community/community#86687] [GitHub Education]
### Hi everyone,

I love GitHub Classroom... and I use it for my C programming class at the University of Hawaii.  One of the features I really got into last year was the autograder (which creates actions).  

For the last couple of weeks, when you edit an assignment and then try to edit a test, the pencil-shaped edit button does nothing.  

I've reproduced this on Safari, Chrome and Edge.

I can add new tests, but I can't edit them.

<img width="524" alt="2024-01-11_22-24-50" src="https://github.com/community/community/assets/10577033/c55f1ec3-8398-41e7-86d7-6fb507bf2d7a">

As a professor, I really depend on these tools and I've invested a lot of time in building out autograded assignments.  I realize there are some changes coming to GitHub Classroom... but you haven't implemented them yet.  

Please don't leave the people who've adopted your technology hanging out to dry.

Very Respectfully,
Mark

**Comment by marknelsonengineer** [comment]
So, this is the problem with community-based support... GitHub is not responding to my original post.  It's been 2 weeks now and the autograder edit buttons are still broken.

As a professor, I really depend on this feature.  It frustrates me that GitHub drives us towards using their technology and when it breaks, they leave us stranded.  

Please, GitHub... I'm using your stuff.  Can you take a look at this and help all of us by fixing the issue or posting a workaround.

**Reply by NachiketUN** [reply]
I agree, it is quite cumbersome to delete and create test cases, just to edit them.

**Comment by nixpad** [comment]
@marknelsonengineer @NachiketUN This should be fixed now! So sorry for the inconvenience and thanks so much for your patience!
